package/tests/defaults.yml

```diff
@@ -0,0 +1,9 @@
+---
+classes:
+  - ..main
+
+parameters:
+  # TODO: remove before merging
+  components:
+    openshift4-service-mesh:
+      version: master
```

Comment:
I'll remove this before merging the PR, this will break the golden tests until we tag version v1.0.0 of the component. Unfortunately, there's no clean way to avoid this with the current `component package compile`.